Fronter/Source/UpdateChecker/UpdateChecker.cpp

```diff
@@ -0,0 +1,216 @@
+#include "UpdateChecker.h"
+#include "VersionParser.h"
+#include <set>
+
+
+bool newerVersion(const std::string& currentVersion, const std::set<std::string>& versions)
+{
+	auto newestVersion = currentVersion;
+	auto changed = false;
+	for (const auto& version : versions)
+	{
+		const auto newestDotPos = newestVersion.find('.');
+		const auto versionDotPos = version.find('.');
+		
+		if (std::stoi(version) > std::stoi(newestVersion))
```

Comment:
I think you meant to extract the left sides here.

Reply:
Doesn't `stoi` do that for us? For example `stoi("1.5F")` should return 1.

Reply:
Hmm, perhaps technically true, but take my comment as an indication that it's unclear to people reading the code.